This patch ensures that the correct version number is used. Since there is no tar-ball,
the script to extract the version number fails and returns UNKNOWN
--- configure.ac.orig	2024-08-13 02:45:05
+++ configure.ac	2024-08-13 02:55:46
@@ -4,7 +4,7 @@
 AC_PREREQ([2.69])
 AC_CONFIG_MACRO_DIR([m4])
 AC_INIT([procps-ng],
-	m4_esyscmd([local/git-version-gen .tarball-version]),
+	@@VERSION@@,
 	[procps@freelists.org],,[https://gitlab.com/procps-ng/procps])
 AM_INIT_AUTOMAKE([foreign 1.11 subdir-objects -Wall -Wno-portability tar-pax no-dist-gzip dist-xz])
 AM_SILENT_RULES([yes])
